frontend/src/components/SignForms/SignInForm/SignComponents/SignLabel.tsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+
+import { Typography } from '@mui/material';
+import { authLabel } from '../FormikReceiver';
+
+const SignLabel = (): any => (
```

Comment:
Please don't use `any` type anywhere, it's not safe enough

Reply:
Already fixed